### PR TITLE
Give vox health regen for poison below 20 damage

### PR DESCRIPTION
**Pull request by PJB3005**
This enables Vox to go about 30 seconds in oxygen and have the damage naturally go away. It's also just a nice species plus for them.

This is done in context of https://github.com/space-wizards/space-station-14/issues/33720

:cl:
- tweak: Vox now have natural poison regeneration, allowing them to go for ~30 seconds in oxygen and heal the damage away.

### CODE REVIEW

**Review by Errant-4 — APPROVED** [review]
Approximately 36s of breathing standard airmix before the damage becomes "irreversible"
Health recovers in up to 1m40s

Total time to crit went from ~~2m~~ 2m12s to 2m20s, that seems negligible.

At first glance this does not seem like it would upset anything else. For a hotfix, I would say let's go. Possibly it can even stay until vox gets an actual design review?

**Review by slarticodefast — APPROVED** [review]
Not sure if this needs to be a hotfix considering the next release is right around the corner, but in case you want to hotfix it, here is the third approval.

**Comment by Errant-4** [comment]
Oh I did not mean "literal" hotfix

**Comment by Errant-4** [comment]
I will merge this in about 12 hours if there aren't any compelling objections, and then we can discuss it during the regular staging review on sunday, before it gets to stable

**Comment by Djungelskog2** [comment]
I really think the better solution is to just give vox masks that they can eat/breathe through. Eating and drinking shouldnt be a challenge that vox have to face and its just a chore. Giving vox NOTHING to combat this issue can arguably feel MORE spacist, just like how we had those crappy nitrogen lockers with a paper sign taped on. myself and other admins have already expressed our thoughts in the discord thread and I believe its not doing anything to solve the current issues of implications of specism, eating/drinking being a chore to play with, and vox already gunning for the theatre masks roundstart because they have that functionality (and no, the solution isnt to nerf the theatre masks).

**Comment by PJB3005** [comment]
I do not necessarily disagree that allowing vox to eat through masks is a bad solution. I just think that no matter what we do this PR is nice to have regardless and doesn't have much future risk associated with it.

**Comment by IProduceWidgets** [comment]
Made #33731 as a potential alternative

**Comment by PJB3005** [comment]
> Why not make them heal all poison damage passively as their gimmick, they're basically vultures that exhale ammonia so it wouldn't be too farfetched imo

Would've been more than a 10 line YAML change.

**Comment by Errant-4** [comment]
Additionally, that would be way more than just a limited QOL feature that mostly just addresses a particular species-specific interaction issue, and should need to be backed up by an actual approved design doc

**Comment by Errant-4** [comment]
For the record, I have taken more accurate, repeated measurements of the crit time on live and with the PR. My new numbers are now validated to be accurate to within* 2 seconds 
(there is a delay of variable length before the first oxygen breath, depending on when in the 4-second breath-cycle the mask is taken off)

The difference in time to crit from full health is just 7.7 seconds. Small enough to be effectively irrelevant

**Pull request merged**
